crates/egui/src/containers/window.rs

```diff
@@ -1213,6 +1240,7 @@ impl TitleBar {
         open: Option<&mut bool>,
         collapsing: &mut CollapsingState,
         collapsible: bool,
+        mut collapsed: Option<&mut bool>,
```

Comment:
has to be `mut collapsed` due to `if let Some() = ` causes the mutable reference to `move` and I needed to use it twice..